### DISCUSSION TITLE
How to see removable drive attached via OTG?

### DISCUSSION THREAD

**Discussion by smartm0use** [AndronixApp/AndronixOrigin#174] [Q&A]
I have attached a pendrive to my phone using an OTG cable. On Android (I have LineageOS 17.1) I can see the content with a file explorer. With Termux (version 0.117) I see that it is mounted:

```/dev/block/vold/public:8,1 /mnt/media_rw/<drive-label> vfat...```

I can access it since the phone is rooted (otherwise I got a permission error).

Now, how to access it from Andronix Ubuntu?
I have installed [Ubuntu 20](https://github.com/AndronixApp/AndronixOrigin/blob/master/Rootfs/Ubuntu20/focal-arm64.tar.gz)...

**Comment by elvisisvan** [comment]
i think you can create symbolic link of the location where pendrive is mounted to where ubuntu is
